starboard/shared/starboard/player/video_dmp_writer.cc

```diff
@@ -71,17 +74,17 @@ SB_ONCE_INITIALIZE_FUNCTION(PlayerToWriterMap, GetOrCreatePlayerToWriterMap);
 
 }  // namespace
 
-VideoDmpWriter::VideoDmpWriter() : file_(kSbFileInvalid) {
+VideoDmpWriter::VideoDmpWriter() : file_(-1) {
   int index = 0;
   std::string file_name;
-  while (!SbFileIsValid(file_)) {
+  while (IsValid(file_)) {
     std::stringstream ss;
     ss << "video_" << index << ".dmp";
     file_name = ss.str();
 
     bool created = false;
```

Comment:
created variable is not used anywhere and can be removed.

Reply:
Done.